dotcom-rendering/src/lib/useIsMyGuardianEnabled.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+import { getEnvironmentClient } from './bridgetApi';
+
+export const useIsMyGuardianEnabled = (): boolean | undefined => {
+	const [isEnabled, setIsEnabled] = useState<boolean | undefined>(undefined);
+
+	useEffect(() => {
+		void getEnvironmentClient()
+			.isMyGuardianEnabled()
+			.then(setIsEnabled)
+			.catch((error) => {
+				setIsEnabled(false);
+				console.log('isMyGuardianEnabled', { error });
```

Comment:
We tend to use `@guardian/libs`’s log

Reply:
setting the subscription to dotcom: these options seem a little out of date
![image](https://github.com/guardian/dotcom-rendering/assets/32312712/92647789-bbf8-4943-abac-095010d0eee7)

Reply:
Do you mean that `dotcom` should now be `webx`? It’s still called @guardian/dotcom-platform